calibration/CTDMOG/CGINS-CTDMOG-13616__20150618.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13616,CC_ptcb1,-0.000225,
 37-13616,CC_ptcb2,0,
 37-13616,CC_ptempa0,154.1205,
-37-13616,CC_ptempa1,-0.0654409,
-37-13616,CC_ptempa2,-4.36388e-07,
-37-13616,CC_wbotc,3.4718e-07,
+37-13616,CC_ptempa1,-6.544089e-02,
+37-13616,CC_ptempa2,-4.363881e-07,
+37-13616,CC_wbotc,3.471849e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT